packages/rest/tests/did.test.ts

```diff
@@ -0,0 +1,44 @@
+import type { Agent } from '@aries-framework/core'
+import type { Server } from 'net'
+
+import request from 'supertest'
+
+import { startServer } from '../src'
+
+import { getTestAgent } from './utils/helpers'
+
+describe('DidController', () => {
+  let app: Server
+  let aliceAgent: Agent
+
+  beforeAll(async () => {
+    aliceAgent = await getTestAgent('Connection REST Agent Test Alice', 3999)
+    app = await startServer(aliceAgent, { port: 3000 })
+  })
+
+  afterEach(() => {
+    jest.clearAllMocks()
+  })
+
+  describe('Get did resolution result by did', () => {
+    test('should give 200 when did resolution record is found', async () => {
+      const did = 'did:key:z6Mkk7yqnGF3YwTrLpqrW6PGsKci7dNqh1CjnvMbzrMerSeL'
+      const response = await request(app).get(`/dids/${did}`)
+
+      expect(response.statusCode).toBe(200)
+      expect(response.body.didDocument.id).toBe(did)
```

Comment:
I think it would be good to check the rest of the did document here, so we know for certain the json serialization correctly serializes the model properties to the other properties (e.g. `context` to `@context`). You can just copy the response once and test if it equals the result

Reply:
Got it